====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0c615ac3d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ff72250bf32d739659c01509aab143f9
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0c6159c700>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ff72250bf32d739659c01509aab143f9
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0c6159c7f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ff72250bf32d739659c01509aab143f9
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
WebDriver version 128.0.6613.137 selected
Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.137/linux64/chromedriver-linux64.zip
About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.137/linux64/chromedriver-linux64.zip
Driver downloading response is 200
Get LATEST chromedriver version for google-chrome
Driver has been saved in cache [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137]
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(104, 'Connection reset by peer')': /session/5db46787f6fd5bed904a431a5bdf3ff4
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f05a149e800>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5db46787f6fd5bed904a431a5bdf3ff4
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f05a149d930>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5db46787f6fd5bed904a431a5bdf3ff4
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f27c6a6c3d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/e36835b7753e32e2685b6351737866b8
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f27c6a6e800>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/e36835b7753e32e2685b6351737866b8
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f27c6a6db40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/e36835b7753e32e2685b6351737866b8
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fc4370>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fb4250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fb4730>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fb4a00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fb49a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8890fb4dc0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f773d0df634e72f1f9ea0d9b5ce51abb
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f14780b8400>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f38400>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f385b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f389a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f38b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f38cd0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.306/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39090>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39270>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39480>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f398d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39a80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39c00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element/f.220B410CA37307096CD783969146A64F.d.AD189B098B96F8ED08AD50025BA19FC2.e.307/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f3a050>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f3a260>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f3a470>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f399f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39de0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1477f39cf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/883e5354dd6d191d59e15f3499b6cb58
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9cac400>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b30430>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b305e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b309a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b30b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b30cd0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.312/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31090>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31270>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31480>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b318d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31a80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31c00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element/f.CAB7A87E410DF2431F1F54078767D2CF.d.E63A2A289CA5C3CEAF15EABAD338C40B.e.313/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b32050>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b32260>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b32470>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b31ed0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b32440>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbaf9b32800>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/3a873dc290821ebad33d9eab527495a9
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9e3c5b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbc880>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbc6d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbcdf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbd000>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9e3ccd0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbc460>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbc1c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3dc9cbcb80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/615de1e2f1382d1e84d6e95934229594
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee38dc940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee3758400>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee37588e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee3758d60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee3758e80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ee38dcbe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e9a1cbfc52464000457a57193451f4a
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3f3ebb8580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/1eb6db3d61f0a20a35d40624c1665311
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3f3ebb8a30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/1eb6db3d61f0a20a35d40624c1665311
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3f3ebb85b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/1eb6db3d61f0a20a35d40624c1665311
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364a583d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364914fd0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364914c40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364917a60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364915540>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe364914b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4a08ec2db85e1b00116f877cba6d65c9
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd6ac400>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd564a30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd564e20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd565060>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd5652a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd565450>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd565750>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd6afbe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd576cb0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd576fe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd577ca0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd577fa0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd5760b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd575a50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feffd5749a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/edda8a2fd9692e1001986665167335a4
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d49c4430>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d484c820>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d484c790>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d484c5e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d484c9a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe2d484ca90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/f1f283a20b48f59aaa9a61091b1245c2
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(104, 'Connection reset by peer')': /session/5d295c9ca418079ce69a0b16e4bf454b
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f52e2431150>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5d295c9ca418079ce69a0b16e4bf454b
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f52e2430df0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5d295c9ca418079ce69a0b16e4bf454b
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'RemoteDisconnected('Remote end closed connection without response')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.446/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7acd090>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.446/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7acc370>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.446/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7acc5e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.434/text
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7accc70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.434/text
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992830>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element/f.AF279C96C8C2078128408527EAC985A0.d.98A61E3DE5BDF1A195C5D9A9DAE908BF.e.434/text
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7993040>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b79932b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7993460>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b79937f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b79939a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7993b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7accc70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7accaf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7acc6d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7993760>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b79935b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7993550>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992ec0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992e60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992ce0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992980>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992140>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f94b7992290>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/aae111a894b9af43f1a0ede772561721
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f474b9b4520>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2435c6ece3ed0ff8ea72c95abd3d6782
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f474b9b7eb0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2435c6ece3ed0ff8ea72c95abd3d6782
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f474b84c0d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2435c6ece3ed0ff8ea72c95abd3d6782
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af673460>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/execute/sync
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af609a20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/execute/sync
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af664b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/execute/sync
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af673040>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/element
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af672f50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/element
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af672c20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1/element
Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af673fa0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1
Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af672fe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1
Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f03af671810>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b3714cbb0e4ebfd247120aa454cba8b1
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
====== WebDriver manager ======
Get LATEST chromedriver version for google-chrome
Get LATEST chromedriver version for google-chrome
Driver [/home/sam/.wdm/drivers/chromedriver/linux64/128.0.6613.137/chromedriver-linux64/chromedriver] found in cache
